tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
```

Comment:
> How could I define `@Bar` set?

Should be:
```
'@Bar' => {
    '@Foo` => true,
    'add_this_rule` => true,
    'remove_this_rule` => false,
},
```
right? Isn't this the current way to do it?

> then, what would be the difference between `@X,@Y` and `@X,-@Y` ?

That's why I think explicitly disabled rules should be ignored in such case.